app/models/proxy_deposit_request.rb

```diff
@@ -5,10 +5,10 @@ class ProxyDepositRequest < ActiveRecord::Base
   belongs_to :receiving_user, class_name: 'User'
   belongs_to :sending_user, class_name: 'User'
 
-  # attribute generic_work_id exists as result of renaming in db migrations.
+  # attribute work_id exists as result of renaming in db migrations.
```

Comment:
Would it make sense to add an `alias_attribute` for `#generic_work_id` as well as a deprectate notice on `#generic_work_id` and `#generic_work_id=`?